web/src/routes/api/leagues/[leagueId]/standings/+server.ts

```diff
@@ -0,0 +1,67 @@
+import { supabase } from '$lib/db/supabase/supabase-client';
+import { error, json } from '@sveltejs/kit';
+
+// TODO: type the return value
+async function getStandings(leagueId: bigint, startWeek: number = 1, endWeek: number = 17) {
+  const { data, error } = await supabase
+    .from('schedule')
+    .select(
+      `
+      team_id, 
+      win_h2h:win_h2h.sum(), 
+      win_median: win_median.sum(),
+      points_for: points_for.sum(),
+      points_against: points_against.sum(),
+      max_points_for: max_points_for.sum(),
+      teams:team_id(id, name)
```

Comment:
Similarly, you shouldn't need to do `teams:team_id(id, name)` here. I think it's suitable to do `teams!inner(name)`. This will give you just the team names, which is what you should be caring about for generating the standings in the post, and the `!inner` just makes sure the query doesn't return a bunch of `NULL` results.